shpec/shpec_shpec.sh

```diff
@@ -86,6 +86,15 @@ line'
       assert equal "$(curl)" "stubbed body"
       unstub_command "curl"
     end
+
+    it "prevents loosing stubbed function"
```

Comment:
It will be done if we can find a solution for the problem mentioned here https://github.com/rylnd/shpec/pull/92#issuecomment-291652555